### ISSUE TITLE
The rustc book appears to be out of date

### ISSUE THREAD

**Issue by str4d** [rust-lang/rust#106723]
### Location

https://doc.rust-lang.org/rustc/linker-plugin-lto.html

### Summary

The above URL does not show the content currently in the `rustc` book source: https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/linker-plugin-lto.md

Specifically, the rendered docs only show the LTO table up to Rust 1.60, whereas the source includes up to 1.65 after #103955 was merged in #104138.

**Comment by Mark-Simulacrum** [comment]
https://doc.rust-lang.org/nightly/rustc/linker-plugin-lto.html is the nightly page; the one you linked is only updated on a new stable release. In this case #104138 merged into 1.67 and so will ship [January 26th](https://forge.rust-lang.org/).
